lib/players/computer.rb

```diff
@@ -0,0 +1,83 @@
+require 'pry'
+
+class Player::Computer < Player
+  #not the best 
+  def move(board)
+    move = side(board)
+    move = corner(board) unless corner(board).nil?
+    move = center(board) unless center(board).nil?
+    move = block_fork(board) unless block_fork(board).nil?
+    1.upto(9) { |i| move =  fork(i, board) unless fork(i, board).nil? }
+    1.upto(9) { |i| move =  block(i, board) unless block(i, board).nil? }
+    1.upto(9) { |i| move = win(i, board) unless win(i, board).nil? }
+    puts "Computer Moves to position #{move}"
+    move.to_s 
+  end
+
+  def win(position, board)
+    win = Game::WIN_COMBINATIONS.any? do |combo|
+      combo.all? do |x|
+        (board.cells[x] == token || x == position - 1) && board.valid_move?(position)
+      end
+    end
+    win ? position : nil
```

Comment:
Inline `if` is generally preferable to a ternary operator that returns nil.

`position if win`